In [165]:
movies_df["US Gross"].mean()

44002085.163744524

In [183]:
movies_df["US Gross"].median()

22019465.5

In [166]:
movies_df["US Gross"].var()

3913166983182700.5

In [167]:
movies_df["US Gross"].std()

62555311.39066211

In [192]:
movies_df["Running Time (min)"].mean()

110.19354838709677

In [190]:
movies_df["Running Time (min)"].median()

107.0

In [191]:
movies_df["Running Time (min)"].var()

406.8697927793207

In [189]:
movies_df["Running Time (min)"].std()

20.171013677535413

In [168]:
movies_df["Production Budget"].mean()

31069171.4484375

In [186]:
movies_df["Production Budget"].median()

20000000.0

In [169]:
movies_df["Production Budget"].var()

1266357235689725.0

In [171]:
movies_df["Production Budget"].std()

35585913.44464443

In [172]:
movies_df["IMDB Rating"].mean()

6.283467202141901

In [185]:
movies_df["IMDB Rating"].median()

6.4

In [173]:
movies_df["IMDB Rating"].var()

1.5682300903199902

In [176]:
movies_df["IMDB Rating"].std()

1.2522899386004784

In [177]:
movies_df["IMDB Votes"].mean()

29908.64457831325

In [184]:
movies_df["IMDB Votes"].median()

15106.0

In [179]:
movies_df["IMDB Votes"].var()

2019386306.1193645

In [180]:
movies_df["IMDB Votes"].std()

44937.58233504963

### Os que tiveram maior variação entre media e mediana foram "US Gross", "Production Budget" e "IMDB Votes", seus desvio padrão ficaram em 62555311.39066211, 35585913.44464443 e 44937.58233504963 respectivamente.

### Conclusão que tiro é que valores gastos e ganhos em filmes são bastante relativos, onde por exemplo atividade paranormal o custo foi baixo em relação aos ganhos e que fica um disparate em relação a filmes semelhantes, respectivamente Votos vão de gostos pessoais por filmes e portanto há bastante variação.